src/server/db_slice.cc

```diff
@@ -1485,24 +1433,13 @@ void DbSlice::PerformDeletion(PrimeIterator del_it, ExpireIterator exp_it, Engin
 
   DbTableStats& stats = table->stats;
   const PrimeValue& pv = del_it->second;
-  if (pv.IsExternal()) {
```

Comment:
make sure we write journal changes after we call PerformDeletion. f.e ExpireIfNeeded